Dockerfile

```
@@ -13,9 +13,13 @@ ENV PATH ${PATH}:${SPARK_HOME}/bin
 
 WORKDIR ${SPARK_HOME}
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    gcc \
+    build-essential \
```

Run succeeded: https://amoy.ai/project/view/qiao/training/pipeline/view/churn-aws/run/10/step/preprocess/logs

```suggestion
    build-essential \
```

`gcc` is included in `build-essential`.